packages/nextjs/src/server/index.ts

```diff
@@ -1,7 +1,15 @@
-export * from './clerkClient';
+/**
+ * Generic exports
+ */
+export { verifyToken, createClerkClient } from '@clerk/backend';
+export type { WebhookEvent, WebhookEventType } from '@clerk/backend';
```

Comment:
After [this PR](https://github.com/clerk/javascript/pull/2365) only the following exported types will be available from the `@clerk/backend` : Organization, Session, User, WebhookEvent, WebhookEventType
We should export all those.

Reply:
What about the API types?

Reply:
Adding a TODO, I will update the types in a 2nd PR right after